<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>615</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>1448</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(238, 238, 238);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="stackedWidgetPage1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item alignment="Qt::AlignHCenter|Qt::AlignTop">
        <widget class="QLabel" name="label_7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>450</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>275</width>
           <height>120</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">alternate-background-color: rgb(119, 118, 123);

    background-color: #16344D; /* Color de fondo */</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../../assets/icono.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>30</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton">
         <property name="styleSheet">
          <string notr="true">border:none;
font:18px;</string>
         </property>
         <property name="text">
          <string>Usuario:</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../assets/iconos/user.png</normaloff>../../assets/iconos/user.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLineEdit" name="lineEditUsuario">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {

    background-color: #f5f5f5; /* Gris más claro cuando se escribe */
    border: none; /* Sin bordes generales */
    border-bottom: 1px solid #000000; /* Solo borde inferior inicial */
    border-radius: 0px; /* Sin bordes redondeados */
    padding: 5px; /* Espaciado interno */
    font: 18px;
QLineEdit {

    background-color: #f5f5f5; /* Gris más claro cuando se escribe */
    border: none; /* Sin bordes generales */
    border-bottom: 1px solid #000000; /* Solo borde inferior inicial */
    border-radius: 0px; /* Sin bordes redondeados */
    padding: 5px; /* Espaciado interno */
    font: 10px;

	margin-right: 8px; /* Margen a la derecha de 10px */
}

/* Al pasar el mouse, se agregan los bordes izquierdo y derecho */
QLineEdit:hover {
    border-bottom: 2px solid #16344D; /* Mantiene el borde inferior */
}

/* Al hacer clic (foco), se mantiene igual que el hover */
QLineEdit:focus {
    border-bottom: 2px solid #16344D;
    background-color: #e0e0e0; /* Gris más claro cuando se escribe */
    box-shadow: 0 0 5px rgba(0, 0, 0, 0.3); /* Sombra opcional al activar */
}

}

/* Al pasar el mouse, se agregan los bordes izquierdo y derecho */
QLineEdit:hover {
    border-bottom: 2px solid #16344D; /* Mantiene el borde inferior */
}

/* Al hacer clic (foco), se mantiene igual que el hover */
QLineEdit:focus {
    border-bottom: 2px solid #16344D;
    background-color: #e0e0e0; /* Gris más claro cuando se escribe */
    box-shadow: 0 0 5px rgba(0, 0, 0, 0.3); /* Sombra opcional al activar */
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_2">
         <property name="styleSheet">
          <string notr="true">border:none;
font:18px;</string>
         </property>
         <property name="text">
          <string>Contraseña:</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../assets/iconos/padlock.png</normaloff>../../assets/iconos/padlock.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLineEdit" name="lineEditPassword">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {

    background-color: #f5f5f5; /* Gris más claro cuando se escribe */
    border: none; /* Sin bordes generales */
    border-bottom: 1px solid #000000; /* Solo borde inferior inicial */
    border-radius: 0px; /* Sin bordes redondeados */
    padding: 5px; /* Espaciado interno */
    font: 18px;
QLineEdit {

    background-color: #f5f5f5; /* Gris más claro cuando se escribe */
    border: none; /* Sin bordes generales */
    border-bottom: 1px solid #000000; /* Solo borde inferior inicial */
    border-radius: 0px; /* Sin bordes redondeados */
    padding: 5px; /* Espaciado interno */
    font: 10px;

	margin-right: 8px; /* Margen a la derecha de 10px */
}

/* Al pasar el mouse, se agregan los bordes izquierdo y derecho */
QLineEdit:hover {
    border-bottom: 2px solid #16344D; /* Mantiene el borde inferior */
}

/* Al hacer clic (foco), se mantiene igual que el hover */
QLineEdit:focus {
    border-bottom: 2px solid #16344D;
    background-color: #e0e0e0; /* Gris más claro cuando se escribe */
    box-shadow: 0 0 5px rgba(0, 0, 0, 0.3); /* Sombra opcional al activar */
}

}

/* Al pasar el mouse, se agregan los bordes izquierdo y derecho */
QLineEdit:hover {
    border-bottom: 2px solid #16344D; /* Mantiene el borde inferior */
}

/* Al hacer clic (foco), se mantiene igual que el hover */
QLineEdit:focus {
    border-bottom: 2px solid #16344D;
    background-color: #e0e0e0; /* Gris más claro cuando se escribe */
    box-shadow: 0 0 5px rgba(0, 0, 0, 0.3); /* Sombra opcional al activar */
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>100</height>
          </size>
         </property>
        </spacer>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="BtnIngresar">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>45</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #16344D; /* Color de fondo */
    border-radius: 2px; /* Bordes redondeados */
    color: white; /* Color del texto */
    padding: 5px 10px; /* Espaciado interno */
    border: none; /* Elimina el borde predeterminado */
    font-size: 22px; /* Tamaño de fuente */
    text-align: Center; /* Alinea el texto a la izquierda */
}

QPushButton:hover {
    background-color: #D9844C; /* Color de fondo al pasar el mouse */
    box-shadow: 0 0 10px rgba(0, 0, 0, 0.5); /* Sombra al pasar el mouse */
    transform: scale(1.02); /* Ligero aumento al pasar el mouse */
    transition: all 0.3s ease-in-out; /* Transición suave */
}

QPushButton::icon {
    padding-right: 5px; /* Espaciado entre el icono y el texto */
}
</string>
         </property>
         <property name="text">
          <string>Ingresar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
